addons/concertina_wire/functions/fnc_handleKilled.sqf

```diff
@@ -19,7 +19,7 @@ TRACE_2("params",_wire,_killer);
 private ["_distance", "_vehicle"];
 
 if (isNull _killer) then {
-    _killer = _wire getVariable ["ace_concertina_wire_lastDamager", objNull];
+    _killer = _wire getVariable [QGVAR(lastDamager), objNull];
     if (isNull _killer) then {
         private _midPoint = ((_wire selectionPosition "start") vectorAdd (_wire selectionPosition "deploy")) vectorMultiply 0.5;
         {
```

Comment:
I was looking at this with my phone and accidentally posted that. Didn't let me delete it on my phone.